Automattic/vip-cli
# Introduce .nvmrc and npm check on publish

## Description
Introduced `.nvmrc` to with the Node version to use to build/execute the CLI.

Updated the documentation to reflect the change and also to include extra information on testing with the minimum supported version.

## Steps to Test

[nvm](https://github.com/nvm-sh/nvm) or multiple node versions are required for the test.

Example:

1. Check out PR.
2. if you have `nvm` installed try doing `nvm use` on the project folder, it should activate Node 14/npm v6
3. Run `npm install` to ensure you have all the new dependencies.
4. Move onto Node 16 (if you have `nvm` run `nvm use 16`)
5. run `npm run publish-please`, you should receive this error 
  ```
  npm: 8.5.0
  Wanted npm version ^6  (>=6.0.0 <7.0.0)
  ```
6. Run `nvm use` (or go back to Node 14)
7. run `npm run publish-please`, you should not see the error and it should start running the tests (and give a different error since we're on a wrong branch)
